Application/src/view/visuals/component/Message.java

```diff
@@ -233,13 +239,35 @@ public void renderDefault(GraphicsContext gc) {
 
 
         //Draw trail for the message, for each instance in the arraylist
-        for (Trail t: trails)
-            gc.drawImage(trail, t.getXcoordinate(), (t.getYcoordinate() + 18), t.getWidth(), t.getHeight());
+        for (int i = 0; i < trails.size()-1; i++){
+
+            // Index each trail
+			// Starts with the second index to fix visual issue with the first cloud sprite
+            Trail t = trails.get(i+1);
```

Comment:
Please dont do this, its confusing anyone else to read this

Reply:
@timjon do what?

Reply:
The thing you are doing in this line

Reply:
Using a for loop and then doing stuff with another index

Reply:
@timjon starting with index 1 instead of index 0 is how i skip drawing the first trail :)